checker/tests/nullness-initialization/Suppression.java

```diff
@@ -1,8 +1,11 @@
+import org.checkerframework.checker.nullness.qual.KeyFor;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
 public class Suppression {
 
     Object f;
 
-    @SuppressWarnings("nullnessnoinit")
```

Comment:
What happened to `nullnessnoinit`? Is it still supported? I guess so, because the checker is named that way. Could you add it to the documentation?

Is this change needed or only follows the new naming? If there is a change in behavior, it should be highlighted in the changelog.